bcipy/helpers/visualization.py

```diff
@@ -139,9 +144,13 @@ def generate_offline_analysis_screen(
                 bbox_inches='tight',
                 format='pdf')
 
+        fig_handles.append(fig)
+
     if show_figure:
         plt.show()
 
+    return fig_handles
```

Comment:
Now that we're returning a value we should update the return type in the method signature. I believe it currently says `-> None`.